testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/credentialstore/AddCredentialStoreOnlineTest.java

```diff
@@ -24,13 +29,26 @@
     private static final String TEST_CREDENTIAL_STORE_NAME2 = "CreaperTestCredentialStore2";
     private static final Address TEST_CREDENTIAL_STORE_ADDRESS2 = SUBSYSTEM_ADDRESS
             .and("credential-store", TEST_CREDENTIAL_STORE_NAME2);
+    private static final String PATH_ADDRESS_NAME = "tmp.path";
+    private static final Address TMP_PATH_ADDRESS = Address.of("path", PATH_ADDRESS_NAME);
 
     private static final String TEST_PASSWORD = "somePassword";
 
+    private static String tmpFilePath;
```

Comment:
why is this necessary? and TemporaryFolder tmp is not enough?

Reply:
Is this just shortcut for tmp.getRoot().getAbsolutePath() ?

Reply:
yes, I want to avoid multiple `tmp.getRoot().getAbsolutePath()`